### DISCUSSION TITLE
[Sega Saturn] External single player receiver

### DISCUSSION THREAD

**Discussion by luisredo** [darthcloud/BlueRetro#852] [Q&A]
![saturn_esp32blueretro](https://github.com/darthcloud/BlueRetro/assets/21347315/4c692a53-c059-43fa-a95d-6e294c98f225)
![front_esp32](https://github.com/darthcloud/BlueRetro/assets/21347315/af12b072-116a-495b-80ab-d5adb45b479d)
![back_esp32](https://github.com/darthcloud/BlueRetro/assets/21347315/c3c0b0ec-23ef-4607-8644-37684dda8cbc)
### BlueRetro firmware version

1.9 sega saturn

### BlueRetro firmware specification

HW1

### BlueRetro firmware variant

System specific

### BlueRetro hardware type

External adapter dongle (1 port only)

### Manufacturer

me

### System used

SEGA Saturn

### Bluetooth controller brand & name

pro nintendo switch, xbox series x, wireless pro game controller

### What is problem? (only list ONE problem per report)

blue retro does not have communication with the sega saturn console, but the controls do pair and from blueretro.io you can connect to the bluetooth

### What did you expect to happen?

I tried following the same tutorial as blueretro from megadrive but with the sega saturn control following the instructions in the connection manual with some 3.3v to 5v and 3.3v level adapters connected directly to the controller adapter but it has no effect on the console, but I can pair controllers.

**Comment by darthcloud** [comment]
I don't know what's instructions you followed but it's clearly not from me.

Mine is here:
https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-Cables-Build-Instructions#cable-schematic-11

You can try to figure out whats wrong base on my schematic.

**Comment by luisredo** [comment]
Ok I'll check the diagram, thanks
